packages/compose/src/hooks/use-dragging/index.js

```diff
@@ -9,10 +9,10 @@ import { useCallback, useEffect, useRef, useState } from '@wordpress/element';
 import useIsomorphicLayoutEffect from '../use-isomorphic-layout-effect';
 
 /**
- * @param {Object}                  props
- * @param {(e: MouseEvent) => void} props.onDragStart
- * @param {(e: MouseEvent) => void} props.onDragMove
- * @param {(e: MouseEvent) => void} props.onDragEnd
+ * @param {Object}                            props
+ * @param {import('react').MouseEventHandler} props.onDragStart
```

Comment:
Apparently typing this argument as `MouseEvent` is wrong because it should be a `React.MouseEvent` and not a raw MouseEvent.

I only changed the places where it affected FocalPointPicker, but the other places in this function might also be typed incorrectly. I think it'd be better to address when we encounter a component that is affected by this, so we can test it better.

Reply:
> I think it'd be better to address when we encounter a component that is affected by this, so we can test it better.

That makes sense, although at the moment some `FocalPointPicker` callbacks expect a `MouseEvent` and some others a `React.MouseEvent` — wouldn't it be better to iron out this discrepancy sooner rather than later (ie. not waiting until we find a component affected by this) ?

Reply:
Yay, `MouseEvent`/`React.MouseEvent` mystery is solved! 💪  9580f90